File: commands/suggest.js
const { botCache, getFromCache } = require('../structures/cache')
const { createId } = require('../utils')

botCache.commands.set('suggest', {
    desc: 'Create a suggestion',
    options: [
        {
            name: 'description',
            type: 'STRING',
            description: 'A detailed description of your suggestions',
            required: true
        }
    ],
    exec: async function (client, interaction) {
        await interaction.reply('Coming soon!')

        const sugId = createId('s_')
    }
})

File: cogs/suggest.py
from discord.ext import commands
from discord_slash import cog_ext, SlashContext
from discord_slash.utils.manage_commands import create_option
from main import allowed_guilds


class Suggest(commands.Cog):
    def __init__(self, client):
        self.client = client

    @cog_ext.cog_slash(name="suggest", description="Suggest an idea for this server.", options=[
        create_option(option_type=3, name="description", description="A detailed description of your idea.",
                      required=True)
    ], guild_ids=allowed_guilds)
    async def suggest(self, ctx: SlashContext):
        await ctx.send(content="This command is coming soon!")


def setup(client):
    client.add_cog(Suggest(client))
